sw/device/silicon_creator/lib/manifest.h

```diff
@@ -258,25 +248,23 @@ typedef struct manifest {
   manifest_ext_table_t extensions;
 } manifest_t;
 
-OT_ASSERT_MEMBER_OFFSET(manifest_t, spx_signature, 0);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, rsa_signature, 7856);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, usage_constraints, 8240);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, spx_key, 8288);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, rsa_modulus, 8320);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, address_translation, 8704);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, identifier, 8708);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, signed_region_end, 8712);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, length, 8716);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, version_major, 8720);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, version_minor, 8724);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, security_version, 8728);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, timestamp, 8732);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, binding_value, 8740);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, max_key_version, 8772);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, code_start, 8776);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, code_end, 8780);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, entry_point, 8784);
-OT_ASSERT_MEMBER_OFFSET(manifest_t, extensions, 8788);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, rsa_signature, 0);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, usage_constraints, 384);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, rsa_modulus, 432);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, address_translation, 816);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, identifier, 820);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, signed_region_end, 824);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, length, 828);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, version_major, 832);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, version_minor, 836);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, security_version, 840);
+OT_ASSERT_MEMBER_OFFSET(manifest_t, timestamp, 844);
```

Comment:
Can we fix this?
https://github.com/lowRISC/opentitan/pull/18487#discussion_r1192480274